File: src/main/java/com/lannstark/lec01/Lec01Main.java
package com.lannstark.lec01;

import java.util.Arrays;
import java.util.List;

public class Lec01Main {

  public static void main(String[] args) {
    long number1 = 10L; // (1)
    final long number2 = 10L; // (2)

    Long number3 = 1_000L; // (3)
    Person person = new Person("윤영진"); // (4)

    final List<Integer> numbers = Arrays.asList(1, 2);
//    numbers = Arrays.asList(3, 4);
    numbers.add(4);
  }

}

File: src/main/kotlin/yoon/lec01/Lec01Main.kt
package yoon.lec01

import com.lannstark.lec01.Person

fun main() {
    var number1 = 10L
    val number2 = 10L

    var number3: Long
    number3 = 5L
    println(number3)

    val number4: Long
    number4 = 10L
    println(number4)

    val numbers: List<Int> = listOf(1, 2)

    var number5: Long? = 10L
    number5 = null

    var person = Person("윤영진")

}